e2e/src/hooks/before.hook.ts

```diff
@@ -1,9 +1,12 @@
 import { Before } from '@cucumber/cucumber';
 import retry from 'async-retry';
+import { PuppeteerScreenRecorder } from 'puppeteer-screen-recorder';
 
 import { BrowserContext } from '../classes/browser-context.class';
 import { RETRY_OPTIONS, MEDIUM_TIMEOUT } from '../utils/timing.utils';
 
+const SAVE_PATH = 'video-rep/test-runs.mp4';
```

Comment:
What do you think about an alternative naming for this variable: e.g.`RECORDING_PATH`?

Reply:
`SAVE_PATH` => `RECORDING_PATH`